services/datamanager/builtin/capture/capture.go

```diff
@@ -52,11 +58,17 @@ type Capture struct {
 
 	collectorsMu sync.Mutex
 	collectors   collectors
-
 	// captureDir is only stored on Capture so that we can detect when it changs
 	captureDir string
 	// maxCaptureFileSize is only stored on Capture so that we can detect when it changs
 	maxCaptureFileSize int64
+
+	// the mongo* struct members are protected by
+	// mongoMU and are either all nil or all non nil
```

Comment:
would it make sense to put them in a struct? (to simplify the `all nil or all non nil` story)

Reply:
done